lib/Elastica/Query/MatchNone.php

```diff
@@ -3,7 +3,7 @@
 namespace Elastica\Query;
 
 /**
- * Match none query. Returns no results.
+ * MatchQuery none query. Returns no results.
```

Comment:
This shouldn't be replaced. See master https://github.com/ruflin/Elastica/blob/master/src/Query/MatchNone.php#L6